IOProject/settings.py

```diff
@@ -28,7 +28,7 @@
     'SECRET_KEY', 'o$e+u#2e8k&5^#g2g3(3ko@hhdl%*mwrso73nl+s4_xfj1f81f')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
Po zmianie na False nie będzie widać logów co trochę utrudnia pracę i trzeba to potem i tak zmieniać lokalnie

Reply:
Tu powinno być pobierane ze zmiennych środowiskowych xd nie kiedy to znikło. Ale w każdym razie ja bym zrobił tak, że pobiera ze zmiennych środowiskowych, a jeżeli nie jest ustawione to daje False. Jeżeli ktoś nie odpala przez docker-compose to może ustawić sobie zmienna środowiskową DJANGO_DEBUG=True i też będzie działać